## Results {.page_break_before}

### CPTAC Model Configuration

The CPTAC dataset provides a unique opportunity to investigate the relationships between multi-omic layers, particularly in predicting the proteome from transcriptome data [Figure 1A]. 
A range of approaches were employed to achieve this, including one-to-one mapping of mRNA transcripts to proteins compared with various regressors: dummy, linear, random forest, and neural networks [Figure 1B]. 
The latter proved to be the most variable, with different architectures yielding diverse predictive accuracies. 
To address this, a NAS was utilized to automate the selection of optimal neural network architectures [Figure 1C]. 
The NAS approach iteratively evaluated various model architectures from a search space to identify the most effective structure for predicting protein quantities from transcriptome data [Figures 1C, 1D]. 
The optimal NAS model was benchmarked against other models, including a dummy regressor, and its performance was evaluated using a 100x2 cross-validation metric across different cancer types. 
The results showed that the optimal NAS model consistently outperformed all other models, including the previously optimal random forest model [Figure 1E].  

![**Comparison of methods for predicting the proteome from the transcriptome**
(A) CPTAC cancer datasets were downloaded from the web and processed to match samples across omic layers. 
(B) Four primary methods were evaluated for predicting the proteome from the transcriptome.
They are a dummy regressor to serve as a benchmark, a random forest regressor, a manually designed neural network, and an optimized neural network chosen via NAS. 
(C) The primary outline of the NAS search space used to identify the optimized neural network. 
(D) The model architecture of the optimized neural network. 
(E) A comparison between the four methods via 100x2 cross validation, indicating the optimized neural network outperforms all the others.](images/fig1_overview.svg){#fig:overview tag="1" width="100%"}
