## Discussion {.page_break_before}

The results of this study demonstrate the potential of NAS in optimizing deep learning models for predicting protein quantities from transcriptome data. 
By automating the selection of optimal neural network architectures, we were able to identify a model that consistently outperformed other approaches, including random forest and linear regression. 
The importance of incorporating biologically relevant features, such as mRNA residual connections and correlated transcripts, was also highlighted. 

The NAS-optimized model's ability to predict protein quantities from transcriptome data with improved accuracy has significant implications for the field of proteogenomics. 
By leveraging the strengths of deep learning and multi-omic data, we can gain a better understanding of the complex relationships between different omic layers. 
This, in turn, can lead to the identification of novel biomarkers and therapeutic targets for diseases. 
We expect that as larger and larger datasets become available for training; 
the approach we outlined here will improve this prediction further.  

The SHAP analysis revealed several distinct categories of transcripts that had varying degrees of impact on protein quantity prediction. 
These findings suggest that the model can capture biologically meaningful patterns in the data, and that the incorporation of correlated transcripts can enhance predictive performance in future models. 
Although the residual connection clearly biased the group of direct transcripts, Figure 5 suggests that different classes of transcripts have different effects on protein quantities. 
For example, snRNA, which functions in the spliceosome to process pre-RNA, have almost no specific impact on protein quantity prediction and are found in the unimpactful, uncorrelated transcript category. 
Messenger RNAs generally have a high impact on predictive function. 
Long non-coding RNAs seem broadly more impactful for a geneâ€™s quantity, whereas the importance of pseudogenes appears to depend on their state of processing. 
Only a small subset of microRNAs (1%) were important for predicting this set of proteins, supporting their role in specific biological regulation.  